.github/workflows/installation-chart.yaml

```diff
@@ -44,9 +44,10 @@ jobs:
           version: ${{ env.HELM_VERSION }}
 
       - name: Set up Kustomize
-        uses: syntaqx/setup-kustomize@v1
+        uses: karancode/kustomize-github-action@v1.3.2
```

Comment:
I see there are a lot of bugs reported but not handled:
https://github.com/karancode/kustomize-github-action/issues

I'm not sure if the failing test related to any of these~

Reply:
Just found the kustomize installation script itself supports setting token by env. It should be work fine from now.